universal-application-tool-0.0.1/app/controllers/PostgresController.java

```diff
@@ -46,4 +46,17 @@ public PostgresController(
             },
             httpExecutionContext.current());
   }
+
+  public Result syncAdd(String name) {
+    Person p = newPerson(name);
+    p.save();
+    return ok("person " + name + " with ID: " + p.id.toString() + " synchronously added.");
+  }
+
+  public Person newPerson(String name) {
+    Person p = new Person();
+    p.id = System.currentTimeMillis(); // not ideal, but it works
+    p.name = name;
+    return p;
+  }
```

Comment:
I think this method can be deleted. There isn't a route or HTML template for it.

Reply:
Removed. This was originally introduced to ease the mocking in unit tests. Since the unit test is removed, this is no longer needed.